src/tree/SubscriptionTreeItem.ts

```diff
@@ -86,8 +87,12 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         if (promptForPassphrase) {
             promptSteps.push(new PassphrasePromptStep());
         }
-        LocationListStep.addStep(wizardContext, promptSteps);
 
+        if (context.advancedCreation) {
+            promptSteps.push(new ImageListStep());
```

Comment:
I'd prefer if you did:
```
if (!context.advancedCreation) {
    wizardContext.image = // whatever image is default
}
```

And added `ImageListStep` to the list of steps regardless of if it's `advancedCreation` or not. It's just a more flexible/readable way of doing the defaults.